dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/netty/RestHttpRequestDecoder.java

```diff
@@ -61,14 +62,16 @@ protected void decode(ChannelHandlerContext ctx, io.netty.handler.codec.http.Ful
 
             } catch (IOException e) {
                 logger.error("", e.getCause().getMessage(), "dubbo rest rest http request handler error", e.getMessage(), e);
+            } finally {
+                // write response
+                try {
+                    nettyHttpResponse.addOutputHeaders(RestHeaderEnum.CONNECTION.getHeader(), "close");
```

Comment:
Why is it necessary to close each connection, turning it into a short-lived connection?